HW_3/task_3.py

```diff
@@ -0,0 +1,28 @@
+"""
+Задайте список из вещественных чисел, округленных до сотых.
+Найдите разницу между максимальным и минимальным значением дробной части элементов.
+
+Ввод: значение типа <list> (либо значения типа <int> – размерность списка)
+Вывод: значение типа <float>
+
+Пример:
+[1.1, 1.2, 3.1, 5, 10.01]
+2.0
+"""
+
+from random import uniform
+
+def get_real_nums (n, first, extreme):
+    return [round(uniform(first, extreme), 2) for i in range(n)]
+
+def find_diff(my_num):
+    num = [round(x - int(x), 2) for x in (my_num)]
```

Comment:
Получить дробную часть числа можно также делением на единицу с остатком.
```suggestion
    num = [round(x % 1), 2) for x in (my_num)]
```